dbt_project.yml

```diff
@@ -15,7 +15,6 @@ test-paths: ["tests"]
 seed-paths: ["seeds"]
 macro-paths: ["macros"]
 snapshot-paths: ["snapshots"]
-docs-paths: ["docs"]
```

Comment:
AFAIK if you remove this, we need to move the `docs/overview.md` file into the `models` directory. This file is used to render the custom welcome page in the docs site

Reply:
let me test it locally to ensure it still works!